parity.gemspec

```diff
@@ -9,7 +9,6 @@ Gem::Specification.new do |spec|
   eos
 
   spec.email = ["dan@thoughtbot.com"]
-  spec.executables = ["development", "staging", "production"]
```

Comment:
Without whitelisting them here (is that what this was doing?), how will the distribution mechanism handle other executables in [bin/](https://github.com/thoughtbot/parity/tree/e6cd1c0e4e002ea193d11fcf9ca3bfd638c9bf75/bin)?